sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample4_RecognizeCustomForms.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    [LiveOnly]
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
```

Comment:
FYI, you don't need `: SamplesBase<FormRecognizerTestEnvironment>` in every part of partial class but it also doesn't hurt.